app/services/rubric_refinement_service.py

```
@@ -0,0 +1,101 @@
+                "You are an expert educational assessment auditor. Your job is to REVIEW the rubric, not to rewrite it. "
+                "Identify clear weaknesses with actionable recommendations. Keep feedback specific and concise.",
+            )
+            .add_message(PromptRole.USER, "Assignment details for context:")
+            .add_json_input(PromptRole.USER, assignment)
+            .add_message(PromptRole.USER, "Rubric to review:")
+            .add_json_input(PromptRole.USER, rubric)
+            .add_message(
+                PromptRole.USER,
+                "Evaluate the rubric on: specificity/measurability, point distribution fairness, coverage of required skills, "
+                "clarity of instructor guidelines, duplication/ambiguity, and appropriate grading flags. "
+                "If criteria are missing or points don’t add up per question, flag them. "
```

Ruff (0.14.8)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)